improved on script with aria2c

albertfougy opened the pull request:
![Screen Shot 2019-03-27 at 9 01 53 PM](https://user-images.githubusercontent.com/17938053/55122936-1ca0c600-50d7-11e9-8dd8-32b81d88f902.png)
 #92 and #88 issues are squashed with the aria2c tool.

philipmat commented:
That looks pretty awesome (also yay VSCode!)

Can we make it so that if the computer doesn't have `aria2c` installed, we fall back on `wget`?

albertfougy commented:
![wgetandAria2c-conditional](https://user-images.githubusercontent.com/17938053/55263263-cb174900-5246-11e9-91e9-1191fb999f10.png)
w/ conditional provided client does not have aria2c installed.

![Screen Shot 2019-03-29 at 5 08 43 PM](https://user-images.githubusercontent.com/17938053/55263302-e7b38100-5246-11e9-8d70-7056f1ddbc7b.png)
 w/ wget exclusive of aria2c 

Besides adding the "fallback" conditional for wget.

1. refactored deprecated string interpolation
2. added shebang
3. split long statements in a couple of places
4. added another flag for wget to show progress from left to right

philipmat commented:
Thank you very much for making the code more readable.
